src/goto-instrument/contracts/contracts.cpp

```diff
@@ -257,12 +257,11 @@ void code_contractst::check_apply_loop_contracts(
   // assigns clause snapshots
   goto_programt snapshot_instructions;
 
+  // track static locals
+  // TODO: restrict to variables declared in the loop's scope
+  // or the functions called in the loop body
```

Comment:
Are we tracking this somewhere (e..g, GitHub issue)?

Reply:
yes we are tracking it